sos/report/plugins/foreman.py

```diff
@@ -297,6 +312,10 @@ def setup(self):
         self.add_file_tags({
             '/usr/share/foreman/.ssh/ssh_config': 'ssh_foreman_config',
         })
+        # if we are on RHEL7 with scl, wrap some Puma commands by
+        # scl enable tfm 'command'
+        if self.policy.dist_version() == 7 and is_executable('scl'):
+            self.pumactl = "scl enable tfm '%s'" % self.pumactl
```

Comment:
We can utilise `SCLPlugin.convert_cmd_scl` method but:
- we dont need to modify `PATH` here
- it might be a bit overkill to use the class at all, if majority of (future) execution will not use SCL at all

Reply:
I see you already use `foreman-puma-status` (good!). If it's easier, we can probably introduce wrappers for other Puma things. I suspect other users may find them useful too.